content/webapp/views/pages/concepts/concept/concept.helpers.ts

```diff
@@ -42,3 +42,251 @@ export const getThemeSectionHeading = (
 
   return `${tabLabel} this ${conceptTypeLabel}`;
 };
+
+type ConceptSection = {
+  display: boolean;
+  label?: string;
+};
+
+type ConceptConfig = {
+  [key: string]: {
+    displayName?: string;
+    showPartOf: boolean;
+    sourcedDescription: boolean;
+    fieldOrArea: ConceptSection;
+    imagesBy: ConceptSection;
+    imagesAbout?: ConceptSection;
+    worksBy: ConceptSection;
+    worksAbout: ConceptSection;
+    contributors: {
+      maxCount: number;
+      label?: string;
+    };
```

Comment:
Right, so this ended up working as it's useful elsewhere! Should we change the booleans (`sourcedDescription`, `showPartOf`) to use this structure as well, would it be more intuitive? (I don't hold strong opinions)

Reply:
(I like the label being there btw, it's a good idea)